settings.php

```diff
@@ -31,9 +31,9 @@
 defined('MOODLE_INTERNAL') || die;
 
 if (has_capability('tool/mergeusers:mergeusers', context_system::instance())) {
-    /**
-     * @var \part_of_admin_tree $ADMIN
-     */
+    //
```

Comment:
Remember that I said you shoundn't have to update existing code, only your new code. I made it explicit.  Leave  this comment as a multiline phpdoc comment, please. It's useful for IDEs. 

And many thanks for all the revision of existing code checkings ;-)